target/linux/ath79/Makefile

```diff
@@ -14,6 +14,8 @@ include $(INCLUDE_DIR)/target.mk
 
 DEFAULT_PACKAGES += \
 	kmod-gpio-button-hotplug swconfig \
-	kmod-ath9k uboot-envtools
+	kmod-ath9k uboot-envtools \
+	kmod-switch-ar8xxx kmod-switch-ip17xx \
```

Comment:
Kmod-switch-ip17xx is only needed on the generic subtarget.

Reply:
done